Add all feature toggles with descriptions at boot

mkllnk opened the pull request:
#### What? Why?

We are adding feature toggles more often now and I saw an opportunity to improve the communication and simplify the removal of feature toggles. We don't need migrations for that any more. And we can add descriptions which are displayed in the admin UI:

![Screenshot from 2023-03-22 14-47-44](https://user-images.githubusercontent.com/3524483/226797196-55f72d95-f387-42bb-b2fe-b6ac41905ff8.png)


#### What should we test?
<!-- List which features should be tested and how.
     This can be similar to the Steps to Reproduce in the issue.
     Also think of other parts of the app which could be affected
     by your change. -->

- Log in as admin.
- Visit /admin/feature-toggle/.
- All feature toggles should be there as before (or with additions).
- Delete all feature toggles.
- Restart Puma: `cd ~/apps/openfoodnetwork/current && sudo systemctl restart puma.service`
- Visit /admin/feature-toggle/.
- All current features should be listed straight away (now deactivated because they were deleted).
- Click on each feature.
- Every feature should have a description at the top.

#### Release notes

<!-- Please select one for your PR and delete the other. -->

Changelog Category: Technical changes

<!-- Choose a pull request title above which explains your change to a
     a user of the Open Food Network app. -->

The title of the pull request will be included in the release notes.


#### Dependencies
<!-- Does this PR depend on another one?
     Add the link or remove this section. -->



#### Documentation updates
<!-- Are there any wiki pages that need updating after merging this PR?
     List them here or remove this section. -->

Update https://github.com/openfoodfoundation/openfoodnetwork/wiki/Feature-toggles

jibees reviewed (approved):
This seems to be pretty cool!

drummer83 commented:
Hi @mkllnk,
Thanks for this PR!

Here is the list of flippers BEFORE staging the PR:
![image](https://user-images.githubusercontent.com/1790176/227940215-56fb4d43-eb54-484f-aa97-a81a56139dff.png)

Then staging the PR, nothing changed at all! 

After deleting all the flippers:
![image](https://user-images.githubusercontent.com/1790176/227940360-a8a2cab1-9707-4888-9e5c-cfbe994fba5b.png)

After restarting the puma with your command provided above (Thanks!):
![image](https://user-images.githubusercontent.com/1790176/227940510-455d943e-0c08-4c1c-bcb6-6645a6116ad2.png)

:question: We can see that the flippers are all deactivated (as expected), BUT the flipper for 'Vouchers' is missing! :question: 

I can confirm, that we now have a little description for each feature. Nice!
![image](https://user-images.githubusercontent.com/1790176/227940847-381afefb-c97a-4c13-ad92-251691dc24e0.png)

I have also tested turning on some of the features, which is working fine:
![image](https://user-images.githubusercontent.com/1790176/227941740-75117f57-d3db-4eed-909a-5651f28c82f9.png)

### Results
- Working as expected apart from the 'Vouchers' missing in the list of flippers.
- Adding feedback-needed to hear your comments if this is an issue or not!
- If that's not an issue - ready to merge!

Thanks!

drummer83 commented:
Oh, and who is in charge of updating the documentation? Any proposals or volunteers?

mkllnk commented:
Yes, this PR is ready to be merged and other PRs with new feature toggles can then rebase and add a description. :tada:

mkllnk commented:
I updated the wiki. It's all in one page now:

* https://github.com/openfoodfoundation/openfoodnetwork/wiki/Feature-toggles

I also updated the link from the code: https://github.com/openfoodfoundation/openfoodnetwork/commit/6cfb86e578853561acd7d501bdb29dbe5e249f91
I'm not sure if there are any other links that need updating. But the old page is still there, linking to the merge page.
